### PR TITLE
MSEDO-332 / MSEDO-333 / MSEDO-338 / MSEDO-340: Justify thumbs Up/Down Proms and Symtoms

### PR DESCRIPTION
**Pull request by DanielFran**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
     "@openapitools/openapi-generator-cli": "1.0.13-4.3.1",
     "@types/file-saver": "2.0.1",
     "@types/jest": "26.0.20",
-    "@types/node": "13.13.4",
+    "@types/node": "14.14.20",
```
Comment (sfdsilva) [inline package.json:109]: Does this affect any already developed code?

Comment (DanielFran) [reply package.json:109]: Nop

File: src/main/webapp/app/shared/shared.module.ts [hunk 2]
```diff
@@ -26,6 +27,7 @@ import { ItemCountComponent } from './pagination/item-count.component';
     SortByDirective,
     SortDirective,
     ItemCountComponent,
+    DefineReasonDialogComponent,
```
Comment (sfdsilva) [inline src/main/webapp/app/shared/shared.module.ts:30]: Not sure if this dialog should be placed where it is needed (even if repeated in multiple modules)... just a note

File: package.json [hunk 3]
```diff
@@ -106,7 +106,7 @@
     "@openapitools/openapi-generator-cli": "1.0.13-4.3.1",
     "@types/file-saver": "2.0.1",
```
Comment (sfdsilva) [inline package.json:107]: Fix warnings of unused variables

File: src/main/webapp/app/shared/shared.module.ts (hunk 2 above)
Comment (DanielFran) [reply src/main/webapp/app/shared/shared.module.ts:30]: It was necessary because it is shared by several modules

Comment (DanielFran) [reply src/main/webapp/app/shared/shared.module.ts:30]: The same dialog us used on all page when justifying feedback

**Review by andrenns — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (DanielFran) [reply package.json:107]: All the warnings are not related with files I changed

**Pull request merged**